include/jemalloc/internal/jemalloc_preamble.h.in

```diff
@@ -47,6 +47,10 @@
 #endif
 #include "jemalloc/internal/hooks.h"
 
+#ifdef JEMALLOC_DEFINE_MADVISE_FREE
+#  define MADV_FREE 8
```

Comment:
We should probably prefix this somehow, to avoid naming conflicts with the builtin, no?

Reply:
JEMALLOC_DEFINE_MADVISE_FREE will only present when MADV_FREE is not found on the system. I assume it doesn't matter in this case?

Reply:
Eh, probably not a big deal.

Reply:
Well, I'll see if we can make it clean with a prefix.